rules/ml/ml_linux_anomalous_sudo_activity.toml

```diff
@@ -0,0 +1,65 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+anomaly_threshold = 75
+author = ["Elastic"]
+description = """
+Looks for sudo activity from an unusual user context. An unusual sudo user could be due to troubleshooting activity or
+it could be a sign of credentialed access via compromised accounts.
+"""
+false_positives = [
+    """
+    Uncommon sudo activity can be due to an engineer logging onto a server instance in order to perform manual
+    troubleshooting or reconfiguration.
+    """,
+]
+from = "now-45m"
+interval = "15m"
+license = "Elastic License"
+machine_learning_job_id = "linux_rare_sudo_user"
+name = "Unusual Sudo Activity"
+risk_score = 21
+rule_id = "1e9fc667-9ff1-4b33-9f40-fefca8537eb0"
+severity = "low"
+tags = ["Elastic", "Linux", "ML"]
+type = "machine_learning"
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1078"
+name = "Valid Accounts"
+reference = "https://attack.mitre.org/techniques/T1078/"
+
+
+[rule.threat.tactic]
+id = "TA0003"
+name = "Persistence"
+reference = "https://attack.mitre.org/tactics/TA0003/"
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1078"
+name = "Valid Accounts"
+reference = "https://attack.mitre.org/techniques/T1078/"
+
+
+[rule.threat.tactic]
+id = "TA0004"
+name = "Privilege Escalation"
+reference = "https://attack.mitre.org/tactics/TA0004/"
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1078"
```

Comment:
Would [Abuse Elevation Control Mechanism](https://attack.mitre.org/techniques/T1548/) be a better fit than T1078 here?

Reply:
I like it, will change

Reply:
looks like T1548 is not accepted by the tests

Reply:
Looks like the issue was that you hadn't updated your branch to pull in the ATT&CK updates